### PR TITLE
Restore plurality

### PR DESCRIPTION
**Pull request by saulakravitz**

### CODE REVIEW

File: input/pagecontent/content-transitions.md
Comment (ptran265) [inline input/pagecontent/content-transitions.md:1]: I noticed that content-transitions page has a note for PatientDecedentFetus that might need revision for clarity:
"Now includes multipleBirth bypassEditFlag and multipleBirthTotal extension which replaces ObservationPluralityVitalRecords"

Comment (ptran265) [reply input/pagecontent/content-transitions.md:1]: maybe could link to the use cases page

Comment (saulakravitz) [reply input/pagecontent/content-transitions.md:1]: The content is incorrect:   Te multipleBirthTotal extension was there in VRCL STU1.1.   The only change is:

Now includes a <a href="StructureDefinition-BypassEditFlag.html"> bypassEditFlag </a>  extension on multipleBirth

Comment (saulakravitz) [reply input/pagecontent/content-transitions.md:1]: Fixed

**Review by ptran265 — APPROVED** [review]

**Pull request merged**